process/smartContract/builtInFunctions/factory.go

```diff
@@ -7,13 +7,10 @@ import (
 	"github.com/mitchellh/mapstructure"
 )
 
-// claimDeveloperRewardsFunctionName is a constant which defines the name for the claim developer rewards function
 const claimDeveloperRewardsFunctionName = "ClaimDeveloperRewards"
-
-// changeOwnerAddressFunctionName is a constant which defines the name for the change owner address function
 const changeOwnerAddressFunctionName = "ChangeOwnerAddress"
-
 const setUserName = "SetUserName"
+const saveKeyValue = "SaveKeyValue"
```

Comment:
the constants could be saved in core and be also used in the tests

Reply:
constants will come from config file - andrei is working on it.

Reply:
ok